climakitae/warming_levels.py

```diff
@@ -301,54 +301,64 @@ def _GCM_PostageStamps_MAIN(self):
-    
-        # Make each plot 
+
+        # Colormap normalization for hvplot -- only for relative humidity!
+        if self.variable2 == "Relative Humidity":
+            cmap = _read_ae_colormap(cmap="ae_diverging", cmap_hex=True)
+
+        # Make each plot
         all_plots = _make_hvplot( # Need to make the first plot separate from the loop
-            data=all_plot_data.isel(simulation=0), 
+            data=all_plot_data.isel(simulation=0),
             clabel=clabel, clim=(vmin,vmax), cmap=cmap, sopt=sopt,
             title=all_plot_data.isel(simulation=0).simulation.item()
         )
-        for sim_i in range(1,num_simulations): 
+        for sim_i in range(1,num_simulations):
             pl_i = _make_hvplot(
-                data=all_plot_data.isel(simulation=sim_i), 
+                data=all_plot_data.isel(simulation=sim_i),
                 clabel=clabel, clim=(vmin,vmax), cmap=cmap, sopt=sopt,
                 title=all_plot_data.isel(simulation=sim_i).simulation.item()
             )
             all_plots += pl_i
-        
-        try: 
-            all_plots.cols(3) # Organize into 3 columns 
+
+        try:
+            all_plots.cols(3) # Organize into 3 columns
             all_plots.opts(title=self.variable2+ ': Anomalies for '+str(self.warmlevel)+'°C Warming by Simulation') # Add title
-        except: 
+        except:
             all_plots.opts(title=str(self.warmlevel)+'°C Anomalies') # Add shorter title
-        
+
         all_plots.opts(toolbar="below") # Set toolbar location
-        all_plots.opts(hv.opts.Layout(merge_tools=True)) # Merge toolbar 
+        all_plots.opts(hv.opts.Layout(merge_tools=True)) # Merge toolbar
         return all_plots
-        
-        
+
+
     @param.depends("reload_data2", watch=False)
     def _GCM_PostageStamps_STATS(self):
-        
-        # Get plot data 
+
+        # Get plot data
         all_plot_data = self._warm_all_anoms
-        if self.variable2 == "Relative Humidity": 
+        if self.variable2 == "Relative Humidity":
             all_plot_data = all_plot_data*100
-        
+
         # Compute stats
         min_data = all_plot_data.min(dim='simulation')
         max_data = all_plot_data.max(dim='simulation')
         med_data = all_plot_data.median(dim='simulation')
         mean_data = all_plot_data.mean(dim='simulation')
-        
-        # Set up plotting arguments 
+
+        # Set up plotting arguments
         width=210
         height=210
         clabel = self.variable2 + " ("+self.postage_data.attrs["units"]+")"
         cmap_name = var_descrip[self.variable2]["default_cmap"]
         cmap = _read_ae_colormap(cmap=cmap_name, cmap_hex=True)
         vmin, vmax, sopt = _compute_vmin_vmax(min_data,max_data)
-        
+
+        if self.variable2 == "Relative Humidity":
+            cmap = _read_ae_colormap(cmap='ae_diverging', cmap_hex=True)
+        else:
+            cmap_name = var_descrip[self.variable2]["default_cmap"]
```

Comment:
Looks like this is a repeat of L352 so this one isn't needed.

Reply:
Thanks @bkg for all of these catches! I've made those changes